pkg/virt-handler/device-manager/device_plugin_base.go

```diff
@@ -53,51 +52,6 @@ type DevicePluginBase struct {
 	deviceName   string
 }
 
-func (dpi *DevicePluginBase) Start(stop <-chan struct{}) (err error) {
```

Comment:
Introduced in the first commit but removed in the second, perhaps we can remove it from the first?